client/containers/PlaceDetails.js

```diff
@@ -0,0 +1,44 @@
+import { connect } from 'react-redux';
+import React, { Component, Fragment } from 'react';
+import placeActions from 'actions/placeActions';
+import PropTypes from 'prop-types';
+
+import Place from 'components/Place/Place';
+import Image from 'components/Image';
+
+const { placeById } = placeActions;
+
+class PlaceDetails extends Component {
+  static propTypes = {
+    placeById: PropTypes.func,
+    match: PropTypes.object,
+  }
+  componentDidMount() {
+    const { params } = this.props.match;
+    this.props.placeById(params.id);
+  }
+  render() {
+    const { props } = this;
+    const { details } = props;
+
+    if (details) {
+      return (
+        <Fragment>
+          <Place place={details} />
+          <Image src={details.img} />
```

Comment:
why not just put `Image` inside `Place`? after all, it's part of `details`

Reply:
Will add image to the Place component